velox/common/memory/SharedArbitrator.cpp

```diff
@@ -419,12 +455,40 @@ uint64_t SharedArbitrator::decrementFreeCapacityLocked(
   return allocatedBytes;
 }
 
+uint64_t SharedArbitrator::getCapacityShrinkTarget(
+    const MemoryPool& pool,
+    uint64_t requestBytes) const {
+  requestBytes = requestBytes == 0 ? pool.capacity() : requestBytes;
+  const auto capacity = pool.capacity();
```

Comment:
Can you move statement: "const auto capacity = pool.capacity();" ahead as it is used by requestBytes setting?